share/src/main/java/io/zensoft/share/service/VacancyRetriever.java

```diff
@@ -0,0 +1,13 @@
+package io.zensoft.share.service;
+
+import io.zensoft.share.model.Vacancy;
+import io.zensoft.share.model.VacancyResponse;
+
+public interface VacancyRetriever {
```

Comment:
rename `VacancyRetriever` to `VacancyRetrieverService`

Reply:
I'll do it. Thank you for your review